tests/ce/test_ce.py

```diff
@@ -0,0 +1,46 @@
+from openfisca_us_data import CE
+
+
+def test_ce_from_2019():
+    """Test the Consumer Expenditure data generating functions for 2019
+
+    Note that the assertions are based on estimates done prior to integration
+    into the openfisca-us-data package and do not exactly match published
+    Consumer Expenditure values. However, only the interview data was used for
+    these estimates (not the diary data) and the documentation warns that
+    users may not be able to match published values due to data disclosure
+    limitations. These were considered sufficiently close for the purposes
+    of simulation.
+    """
+    # 2019 Constants -------------------------------------------------------------
+    HHS_IN_US = 122.8e6
```

Comment:
Is this also from the CE? If so would it be CUs?

Reply:
I guess it's from out of the blue. I googled number of households in the US, just to demonstrate that we're in the ballpark for annual CO2. I can remove it, rename it, ..., your call.

Reply:
This works, could you just add the source?

Reply:
left some more documentation. You can see in the calculations that we're multiplying a per_cu quantity with number of households, which is suggestive at least